### PR TITLE
Goal tile

### PR DESCRIPTION
**Pull request by emilyb7**
Implements goal tile component
Goal tiles are shown on goals list, updated dynamically based on state (which is currently hardcoded in reducers/index.js)
+ Lots of styling fixes
Relates to #96

### CODE REVIEW

**Comment by des-des** [comment]
Ok I am not working this eve so ill get to this first thing tomorrow

File: app/components/goal-tile.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const tile = props => {
+
+  const goal = props.goal;
+  const pathAvatar = `./app/public/images/avatars/${goal.avatar}.png`;
+
+  const style = {
+    background: 'url(./app/public/images/list_box.png)',
```
Comment (des-des) [inline app/components/goal-tile.jsx]: Why not in stylesheet?

File: app/components/goals-list/goals-list.jsx [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
+      key={ goal.id }
+    >
+      <GoalTile
+        goal={ goal }
+      />
+    </li>
   });
 
+  const dynamicStyle = {
+    height: Math.max(window.innerHeight - 90, goals.length * 108 + 115),
```
Comment (des-des) [inline app/components/goals-list/goals-list.jsx]: Yup, this is an example of where you can be inlining your styles

File: app/components/goals-list/goals-list.jsx [hunk 3]
```diff
@@ -1,13 +1,28 @@
...
+  };
+
   return (
-    <div className="page goalsList">
+    <div
+      className="page goalsList"
```
Comment (des-des) [inline app/components/goals-list/goals-list.jsx]: Hmm, you need to choose a style. Some of your components you are putting props on new line, and some you are not... Does that make sense?

File: app/reducers/index.js [hunk 4]
```diff
@@ -10,16 +10,45 @@ const defaultState = {
...
+    {
+      id: 2,
+      name: 'Be cool',
+      created: 1488984810658,
+      avatar: 'pepper',
       status: 0,
+      currentRating: 10,
       ratings: [],
     },
+    // {
```
Comment (des-des) [inline app/reducers/index.js]: :fire:

File: app/scss/_goalTile.scss [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+.goalTile {
+  position: relative;
+  border-radius: 6px;
+  display: flex;
+  flex-direction: row;
+  align-self: center;
+  width: 90%;
+  max-width: 400px;
+  height: 98px;
+  z-index: 0;
```
Comment (des-des) [inline app/scss/_goalTile.scss]: Is this not default?

File: app/scss/_goalTile.scss [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+  align-items: flex-end;
+  top: 20px;
+}
+
+.goalTile_rating {
+  position: absolute;
+  right: 10px;
+  top: 50px;
+  width: 31px;
+  height: 31px;
```
Comment (des-des) [inline app/scss/_goalTile.scss]: Can you check though this css and check if some of these numbers would be better off in vars

File: app/components/goals-list/goals-list.jsx (hunk 3 above)
Comment (emilyb7) [inline app/components/goals-list/goals-list.jsx]: yeah i get your point. I think it looks better on one line, so I've tidied this up. Only there are some cases where there are a lot of props and then it makes it far more readable to have the things on different lines. Can I not do a mixture in some cases?

File: app/reducers/index.js (hunk 4 above)
Comment (emilyb7) [inline app/reducers/index.js]: If can live with this, I would like to leave the commented out goals in here. Obviously won't be hardcoding any of the goals into the state in the real version. This makes it easier for me to test what the app looks like with different numbers of goals!

File: app/scss/_goalTile.scss (hunk 6 above)
Comment (emilyb7) [inline app/scss/_goalTile.scss]: So i managed to create one more variable :D

File: app/reducers/index.js (hunk 4 above)
Comment (des-des) [inline app/reducers/index.js]: Ok, you and @shiryz should work together to have a better way of injecting mock data, once you have the server and the client working together

**Pull request merged**